server/src/com/cloud/network/vpc/VpcManagerImpl.java

```diff
@@ -309,6 +314,19 @@ public void doInTransactionWithoutResult(final TransactionStatus status) {
                     }
                     createVpcOffering(VpcOffering.redundantVPCOfferingName, VpcOffering.redundantVPCOfferingName, svcProviderMap, true, State.Enabled, null, false, false, true);
                 }
+
+                if (_vpcOffDao.findByUniqueName(VpcOffering.routedVPCOfferingName) == null) {
+                    s_logger.debug("Creating default routed VPC offering " + VpcOffering.routedVPCOfferingName);
```

Comment:
Please wrap this log statement in a ``isDebugEnabled()`` check to avoid unnecessary string concatenation when ``DEBUG`` logging is not enabled.